Add Kit Team Invite Email (#27)
srobo/volunteer-emails: SR2020/2020-02-19-kit-team-invite.md

---
to: All Volunteers
subject: Invitation to the Kit Team!
---

Hello!

Are you interested in helping to develop, maintain or support the Student Robotics kit?  If so, we would like to invite you to become part of the Student Robotics "Kit Team".

## What is the Kit Team?

> In short, we, the Kit Team, provide the "kit as a service" to the other half of Student Robotics -- the Competition Team.  You can read more about what the kit team is and our mission [here].

Reviewer: ```suggestion
In short, we, the Kit Team, provide the "kit as a service" to the other half of Student Robotics -- the Competition Team.  You can read more about what the kit team is and our mission [here](https://srobo.github.io/kitbook/kit-team/).
```

## How can I get involved?

> We've only existed since the beginning of this academic year, and we are now looking to grow beyond just being a committee of 5!  We've split our activites up into several 'project teams'.  You can read about the different teams that we are planning on having [here], but for now we are starting with the support, infrastructure and assets teams.  If you would like to join one of these teams, please fill in [this form] and we will get back to you as soon as we can.

Reviewer 2: ```suggestion
We've only existed since the beginning of this academic year, and we are now looking to grow beyond just being a committee of 5!  We've split our activities up into several 'project teams'.  You can read about the different teams that we are planning on having [here], but for now we are starting with the Support, Infrastructure and Assets teams.  If you would like to join one of these teams, please fill in [this form] and we will get back to you as soon as we can.
```

> We've only existed since the beginning of this academic year, and we are now looking to grow beyond just being a committee of 5!  We've split our activities up into several 'project teams'.  You can read about the different teams that we are planning on having [here], but for now we are starting with the Support, Infrastructure and Assets teams.  If you would like to join one of these teams, please fill in [this form] and we will get back to you as soon as we can.

Reviewer: ```suggestion
We've only existed since the beginning of this academic year, and we are now looking to grow beyond just being a committee of 5!  We've split our activities up into several 'project teams'.  You can read about the different teams that we are planning on having [here](https://srobo.github.io/kitbook/kit-team/), but for now we are starting with the Support, Infrastructure and Assets teams.  If you would like to join one of these teams, please fill in [this form](https://forms.gle/QGX7ZBCEbxoiiL9e9) and we will get back to you as soon as we can.
```

## Why should I get involved?

> By joining one of our project teams, you will be helping competition participants access and use the kit.  In the process you'll develop your own team-working, management and logistical skills, as well as seeing your efforts rewarded by a plethora of interesting robots built by teams.

Reviewer 2: ```suggestion
By joining one of our project teams, you will be helping competitors access and use the kit to learn engineering skills and ultimately build their robot. In the process you'll develop your own team-working, management and logistical skills, as well as seeing your efforts rewarded by a plethora of interesting robots built by teams.
```
